docker/pythonsysds.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM openjdk:8-alpine
```

Comment:
yes, i fought through it ... now 500MB image instead of 1-2GB. There is more things that could be done to decrease the size. but not something i will work on now